be2-scala/src/test/scala/ch/epfl/pop/pubsub/graph/AnswerGeneratorSuite.scala

```diff
@@ -0,0 +1,143 @@
+package ch.epfl.pop.pubsub.graph
+
+import akka.actor.{Actor,ActorSystem,Props}
+import akka.actor.typed.ActorRef
+import akka.pattern.AskableActorRef
+import akka.testkit.{ImplicitSender,TestKit,TestProbe}
+import akka.util.Timeout
+import ch.epfl.pop.model.network.{JsonRpcRequest,JsonRpcResponse,ResultObject}
+import ch.epfl.pop.model.network.method.message.Message
+import ch.epfl.pop.pubsub.graph.validator.SchemaValidatorSuite._
+import ch.epfl.pop.pubsub.graph.validators.RpcValidator
+import org.scalatest.{BeforeAndAfterAll,FunSuiteLike,Matchers}
+import util.examples.MessageExample
+import scala.concurrent.duration.FiniteDuration
+
+class AnswerGeneratorSuite extends TestKit(ActorSystem("Test")) with FunSuiteLike with ImplicitSender with Matchers with BeforeAndAfterAll  {
+
+  final val pathCatchupJson = "../protocol/examples/query/catchup/catchup.json"
+  final val pathPublishJson = "../protocol/examples/query/publish/publish.json"
+  final val pathBroadcastJson = "../protocol/examples/query/broadcast/broadcast.json"
```

Comment:
you could have a final val "../protocol/examples/..." to avoid code copy and just add the file name behind (detail)